doc/api/process.markdown

```diff
@@ -873,6 +883,38 @@ a diff reading, useful for benchmarks and measuring intervals:
     }, 1000);
 
 
+## process.send(message[, sendHandle])
+
+* `message` {Object}
+* `sendHandle` {Handle object}
+
+When node is spawned with an IPC channel attached, it can send messages to its
+parent process using `process.send()`. Each will be received as a
+['message'](child_process.html#child_process_event_message)
```

Comment:
@thefourtheye the comment on #2142 was from me. It wasn't about having the link at the bottom of the page, just that the URL needed to be relative (without the https://iojs.org/api/ part). It is fine to have it inline.